Scripts/script-RACopyFrom.yml

```diff
@@ -0,0 +1,54 @@
+commonfields:
+  id: RACopyFrom
```

Comment:
I suggest renaming to RemoteaccessCopyFrom. Much more descriptive and not that long